### PR TITLE
this.x = assignments in namespace functions make properties mutable

### LINKED ISSUE #374: Namespace properties converted as const, should be let
See https://github.com/PolymerElements/iron-overlay-behavior/blob/__auto_generated_3.0_preview/iron-scroll-manager.js#L101

These properties are assigned elsewhere in the file, which throws.

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #374

### CODE REVIEW

**Comment by e111077** [comment]
I think your link is pointing to the wrong bug?

**Comment by e111077** [comment]
This _looks_ right to me but I feel uncomfortable being the only LGTM in this PR.

**Comment by justinfagnani** [comment]
Oops. should be #374

File: src/passes/rewrite-namespace-exports.ts [hunk 1]
```diff
@@ -139,6 +139,10 @@ class RewriteNamespaceExportsPass {
     const fullyQualifiedName = memberPath.join('.');
     this.namespaceNames.add(fullyQualifiedName);
 
+    // Note: in other place in this file where we decide between let and const
```
Comment (FredKSchott) [inline src/passes/rewrite-namespace-exports.ts]: 👍

**Pull request merged**